connectors/sink-feishu/internal/bot.go

```diff
@@ -18,30 +18,216 @@ import (
 	"crypto/hmac"
 	"crypto/sha256"
 	"encoding/base64"
+	"encoding/json"
+	"errors"
 	"fmt"
-	"github.com/linkall-labs/cdk-go/log"
-	"github.com/tidwall/gjson"
 	"time"
 
 	v2 "github.com/cloudevents/sdk-go/v2"
+	"github.com/cloudevents/sdk-go/v2/types"
+	"github.com/go-resty/resty/v2"
+	"github.com/linkall-labs/cdk-go/log"
+	"github.com/tidwall/gjson"
 )
 
-func (f *feishuSink) sendTextToFeishuBot(e *v2.Event) error {
-	t := time.Now()
+type messageType string
+
+const (
+	xChatGroupID = "xvfeishuchatgroup"
```

Comment:
this is feishu sink, is it cant be modified to "xvchatgroup"

Reply:
The reason I added the prefix aims to clarify when users write subscriptions (users may use the different sink in subscription.)

Reply:
maybe you are right, the problem needs to be resolved by vanus but not by sink